teuthology/kill.py

```diff
@@ -187,6 +189,7 @@ def kill_processes(run_name, pids=None):
     if len(to_kill) == 0:
         log.info("No teuthology processes running")
     else:
+        survivors = list()
```

Comment:
Can I ask why `list()` instead of `[]`. I don't mind `list()`, just curious.

Reply:
Good question! It wasn't actually intentional, and `[]` is faster, so I'll replace it :)